### PR TITLE
Add Click Handlers as Props and Organize Props

### LINKED ISSUE #130: Pokemon Click Handlers
## Expected Behavior

If you click a pokemon card on the home page or my poke list page, it should only display that pokemon in the sidebar. If you click a pokemon on the game page it should take you to the my poke list page and display it in the sidebar. 

## Actual/Current Behavior

When a pokemon is clicked on the game page, it does not take the user to the my poke list.

### PR DESCRIPTION
**Pull request by krferrell**
## Changes
1. Removed unnecessary props and alphabetized the remaining 
2. Condensed props into a click handler function and passed it as a prop

## Purpose
The allows for different click handlers to be called depending on where the card is being rendered. The code cleanup is for organization and readability. 

## Approach
Condensed some props into two functions then passed them as props. This allowed us to not have to pass as many props. 

Closes #130

### CODE REVIEW

File: pokeapp/src/components/PokemonCard.js [hunk 1]
```diff
@@ -86,58 +71,54 @@ const PokemonCard = ({
...
-        <div style={cardImage}>
-          {/* put buttons in here */}
-          <div className="card-btns-container">
-            <div className={applyStyles(`poke-value`)}>
-              <img src={PokeDollarIcon} alt="poke dollar image" />
-              {pokemon.value}
-            </div>
+      </div>
+
+      {/* Card image starts */}
```
Comment (Lap343) [inline pokeapp/src/components/PokemonCard.js:80]: trying to visualize it in my head

**Review by kmausolf — COMMENTED** [review]
GREAT cleanup PR.  For alphabetizing your props, I award the team with a whopping 3000 points each!

**Review by Lap343 — APPROVED** [review]
Nice code. Like you said mostly props and small styling changes. Cant wait to play what I think is your gatcha game lol

**Review by JorgeLVilla — APPROVED** [review]
Looking good Kenny!!

File: pokeapp/src/components/SidebarNav.js [hunk 2]
```diff
@@ -1,14 +1,14 @@
...
+export const SidebarNav = ({ setIsBadgeSideBar, setBadgeBtnActive1, setBadgeBtnActive2, badgeBtnActive1, badgeBtnActive2 }) => {
   return (
     <>
       <div
         className={badgeBtnActive1 ? "badge-btn-left active" : "badge-btn-left"}
         onClick={() => {
-          setbadgeBtnActive1(true);
-          setbadgeBtnActive2(false);
+          setBadgeBtnActive1(true);
+          setBadgeBtnActive2(false);
```
Comment (BlackCubes) [inline pokeapp/src/components/SidebarNav.js:11]: Good fix

**Review by BlackCubes — APPROVED** [review]
Everything looks really good, structured, and organized!

**Pull request merged**